kong/global.lua

```diff
@@ -182,17 +182,21 @@ function _GLOBAL.init_worker_events()
                  and configuration.prefix
                  or require("pl.path").abspath(ngx.config.prefix())
 
+  local max_payload_len = configuration
```

Comment:
should add entry in `conf_loader/init.lua`

Reply:
done